### PR TITLE
Hibernate order solution

### PR DESCRIPTION
**Pull request by mkazarian-1**

### CODE REVIEW

File: src/main/java/mate/academy/Main.java [hunk 1]
```diff
@@ -44,12 +55,49 @@ public static void main(String[] args) {
...
+                INJECTOR.getInstance(AuthenticationService.class);
+        try {
+            System.out.println(authenticationService
+                    .register("Mate@gamil.com", "12345qwert"));
+            System.out.println(authenticationService
+                    .register("Academy@gamil.com", "qwert"));
+            System.out.println(authenticationService
+                    .register("LOLIPOP123f@gamil.com", "098765dfde"));
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/Main.java:78]: bad practice to throw RuntimeException

File: src/main/java/mate/academy/dao/OrderDao.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/dao/OrderDao.java:8]:
```suggestion
```

File: src/main/java/mate/academy/model/Order.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToMany
+    private List<Ticket> tickets;
+
+    private LocalDateTime orderDate;
+
+    @ManyToOne
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/model/Order.java:25]: add Lazy

File: src/main/java/mate/academy/service/impl/OrderServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order newOrder = new Order();
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/service/impl/OrderServiceImpl.java:23]:
```suggestion
        Order order = new Order();
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/Main.java (hunk 1 above)
Comment (mkazarian-1) [reply src/main/java/mate/academy/Main.java:78]: it's just code for testing, so I didn't want to waste time

**Review by okuzan — APPROVED** [review]

**Pull request closed without merging**